examples/labels.py

```diff
@@ -0,0 +1,108 @@
+from typing import Any, Dict, List, Callable
+import pandas as pd
+
+
+class FeatureDefinition:
+    # Implement class logic
+    # TODO: Add a very specific type hint to the rule object:
+    # see: https://docs.python.org/3/library/typing.html#annotating-callable-objects
+    def __init__(self, label: str, rule: Callable[[Dict[str, Any]], bool]):
```

Comment:
Alright. Now we need to add [docstrings](https://www.geeksforgeeks.org/python-docstrings/#) to our class methods and functions. Take a look at "indentation in docstrings" on that page for an example of adding doc strings to a class.

These are necessary as user's IDEs will parse them and convert to documentation for end users of our library.